Shared/RegionList.swift

```diff
@@ -67,8 +67,7 @@ struct RegionList: View {
 }
 
 struct CountryList_Previews: PreviewProvider {
-    @State static var isRefreshAlertPresented = false
```

Comment:
Please revert these changes to avoid having unrelated changes in the PR

Reply:
Okay 👍